.snyk

```diff
@@ -0,0 +1,23 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.25.0
+# ignores vulnerabilities until expiry date; change duration by modifying expiry date
+ignore:
+  SNYK-JAVA-ORGYAML-3152153:
+    - '*':
+        reason: >-
+          The MGM CLI plugin accepts YAML input only in order to generate a
+          group policy file. This group policy file is then separately used to
+          assemble a CPI to be uploaded to a Corda cluster. As a result, the
+          Corda platform is not exposed to this vulnerability.
```

Comment:
I do not think `MGM CLI plugin` has anything to do with this Git Hub repository. Therefore proposing to change:
```suggestion
          We are not exposed because we don’t accept yaml based input in any of the components affected.
```

Reply:
Updated with your suggestion thanks @vkolomeyko